### DISCUSSION TITLE
Can I install Edgeshark on Windows?

### DISCUSSION THREAD

**Discussion by Dhowell560** [siemens/edgeshark#59] [Q&A] [answered]
Hello,

I am running Docker Desktop on Windows 10 Enterprise.  Docker is configured to use the WSL 2 backend with the desktop-windows builder enabled.  I run my container on this same machine.  The base OS for the image is Windows Server 2019.  Can I install Edgeshark on this system?  Can you provide instructions please?

Thank you.

**Comment by thediveo** [comment, marked as answer]
Deploying the services:
1. start a WSL2 bash shell
2. copy and paste this command (from the README):
   ```bash
   wget -q --no-cache -O - \
     https://github.com/siemens/edgeshark/raw/main/deployments/wget/docker-compose-localhost.yaml \
     | docker compose -f - up
   ```

Navigate to http://localhost:5001

To setup capture:
1. ensure you have Wireshark installed on Windows, if you want to capture container traffic.
2. download the Wireshark cshargextcap plugin installer, https://github.com/siemens/cshargextcap/releases, unzip it, and run the unzipped setup exe.
